lib/matplotlib/colors.py

```diff
@@ -1992,3 +1994,49 @@ def from_levels_and_colors(levels, colors, extend='neither'):
 
     norm = BoundaryNorm(levels, ncolors=n_data_colors)
     return cmap, norm
+
+
+def brighten_color(color, frac):
```

Comment:
to :bike: :house: the name a bit, should this be `adjust_color` or `adjust_tint` or something like that?  I almost left a comment after quickly skimming the code as "do we need to add a darken color too?" (and then I read the docstring and left this comment instead).

Reply:
Does a function in the module `colors` really need to have `color` as part of its name?

Reply:
adjust_lightness?  Opens up to adjust saturation.